src/sentry/api/exceptions.py

```diff
@@ -56,6 +56,17 @@ def __init__(self, organization):
         super().__init__(loginUrl=reverse("sentry-auth-organization", args=[organization.slug]))
 
 
+class MemberDisabledOverLimit(SentryAPIException):
+    status_code = status.HTTP_401_UNAUTHORIZED
+    code = "member-disabled-over-limit"
+    message = "Must upgrade plan"
```

Comment:
This definitely shouldn't be in sentry right?

Reply:
its fine to put this in sentry but remove notations of upgrade plan/etc - make the code what you use to message to user in UI, and the message can be something like "Organization has hit member capacity." or similar.

Reply:
@EvanPurkhiser Moving this to getsentry adds a lot of complexity. First off, we'd need to build hooks to throw this error. And we'd also need hooks where [we consume this on the front end](https://github.com/getsentry/sentry/blob/6842ad5518d60b3ddc0af512d040b04ac5d1e17e/static/app/api.tsx#L92-L94). Given the level of sensitivity with auth code, I think it's safer and acceptable to keep relevent auth code in Sentry.

@dcramer good suggestion on the message, that makes it more OSS friendly.